<launch>
  <arg name="name" default="concat_velodyne" />
  <arg name="cloud_in1" default="velodyne_front/cloud" />
  <arg name="cloud_in2" default="velodyne_rear/cloud" />
  <arg name="cloud_out" default="velodyne/cloud" />
  <arg name="target_frame" default="vehicle_1/base_link" />
  <arg name="nodelet_manager" default="velodyne_front/nodelet_manager" />

  <node pkg="nodelet" type="nodelet"
    args="load lidar/ConcatenateTwoClouds $(arg nodelet_manager)"
    name="$(arg name)" output="screen">
    <remap from="cloud_in1" to="$(arg cloud_in1)" />
    <remap from="cloud_in2" to="$(arg cloud_in2)" />
    <remap from="cloud_out" to="$(arg cloud_out)" />
    <param name="target_frame" value="$(arg target_frame)" />
  </node>
</launch>